recipes/sv2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.4.1" %}
+{% set sha256sum = "16fdc9e59f837549f149906fe4e771da40265e52d165460ecf6ac8949df7397b" %}
+
+package:
+  name: sv2
+  version: {{ version }}
+
+source:
+  fn: sv2-{{ version }}.tar.gz
+  url: https://github.com/dantaki/SV2/archive/sv2v1.4.1.tar.gz
+  sha256: "{{ sha256sum }}"
+  patches:
+  - sv2.patch
+
+build:
+  skip: True # [py3]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+
+requirements:
+  build:
+  - python
+  - cython
+  - numpy
+  - pandas
+  - pybedtools
+  - pysam
+  - wget
+  run:
+  - python
+  - cython
+  - numpy
+  - pandas
+  - pybedtools
+  - pysam
+  - wget
+
+test:
+  run:
+  - sv2 --help
+  - sv2train --help
+
+about:
+  home: https://github.com/dantaki/SV2
+  summary: Support Vector Structural Variation Genotyper
+  license: MIT
```

Comment:
please add `license_file: LICENSE`

Reply:
Thanks!

Reply:
By the way, is it possible to somehow access the conda package created by CI?